cpp/build-support/update-flatbuffers.sh

```diff
@@ -23,16 +23,18 @@
 CWD="$(cd "$(dirname "${BASH_SOURCE[0]:-$0}")" && pwd)"
 SOURCE_DIR=$CWD/../src
 FORMAT_DIR=$CWD/../../format
-FLATC="flatc -c --cpp-std c++11"
+FLATC="flatc --cpp --grpc --cpp-std c++11"
```

Comment:
I do not think we should generate gRPC extensions here

Reply:
I agree with that. The example service is a good extension, but I don't think it should be part of the core IR right now.

Reply:
One practical reason is that some important languages (such as Python) cannot generate code for services which will cause flatc to fail.